gapis/service/path/path.go

```diff
@@ -651,6 +651,13 @@ func (n *Command) FramebufferAttachmentsAfter() *FramebufferAttachments {
 	}
 }
 
+func (n *Command) FramebufferAttachmentAfter(index uint32) *FramebufferAttachment {
```

Comment:
nit: it would be great that these two adjacent functions could get quick annotations as others!

Reply:
Yes, all public go functions should have a comment. The comment should be in the godoc format (i.e. start with the name of the method).